apps/stripe/src/modules/saleor/saleor-webhook-responses.ts

```diff
@@ -0,0 +1,95 @@
+import { GetConfigError, MissingConfigError } from "@/modules/app-config/app-config-errors";
+import { SaleorApiUrl } from "@/modules/saleor/saleor-api-url";
+
+export class SuccessWebhookResponse {
+  statusCode = 200;
+}
+
+export class ErrorWebhookResponse {
+  statusCode = 500;
+}
+
+export class GetConfigErrorResponse extends ErrorWebhookResponse {
+  error: InstanceType<typeof GetConfigError>;
+  message = "App is not configured - error while getting config";
```

Comment:
This message goes to external service so I would only say "app is not working", which is true. We dont know if its configured - maybe it was, but not DB is not responding. We probably shouldnt elaborate on DB too.

Sentry on the other hand will have details